anchor/anchors.go

```diff
@@ -76,3 +76,22 @@ func (a Anchor) SubType() string {
 func (a Anchor) Value() []string {
 	return a.value
 }
+
+// GetSources returns the anchors which identify how and when an attribute value was acquired.
+func GetSources(anchors []*Anchor) (sources []*Anchor) {
+	return filterAnchors(anchors, AnchorTypeSource)
+}
+
+// GetVerifiers returns the anchors which identify how and when an attribute value was verified by another provider.
+func GetVerifiers(anchors []*Anchor) (sources []*Anchor) {
+	return filterAnchors(anchors, AnchorTypeVerifier)
```

Comment:
Since we already have _Type_ as a string on each Anchor struct (as taken from the certificate), can't we just filter on that?  What's the additional _type_ enum for?

Reply:
*Update*, I'm a bit wrong here, we've just talked this through....

The _anchor type_ comes to us as an 'extensionOId' e.g. '1.3.6.1.4.1.47127.1.1.1'.  We are not given a nice, neat String.  The String's are introduced by the SDK to make the OId more meaningful.

The apprach here matches the Java SDK - map the OId to an Enum as a convenient way to convert that OId into a nice string.  If we ever get an OId we don't know about, we map it to _unknown_.

We might to want to improve on this so I've created SDK-683, but for now the work done here is good to go.